pkg/addons/registry/registry.go

```diff
@@ -101,3 +105,27 @@ func (r *Registry) ChartLocation() string {
 	}
 	return strings.Replace(Metadata.Location, "proxy.replicated.com", r.ProxyRegistryDomain, 1)
 }
+
+// IsRegistryHA checks if the registry has been configured for HA by looking for the
+// REGISTRY_STORAGE_S3_ACCESSKEY environment variable in the docker-registry container.
```

Comment:
why not simply check if the replicas are > 1?

Reply:
nevermind, that happens after the upgrade, not the migration.

Reply:
is this env var added after upgrade or migration?

Reply:
Good call. Thanks!